packages/tinacms/src/internalClient/index.ts

```diff
@@ -425,7 +427,7 @@ mutation addPendingDocumentMutation(
     return !!(await this.getUser())
   }
 
-  async onLogout() {
+  async logout() {
```

Comment:
would it make sense to call the event handlers from these functions instead of in Nav & LogoutPage?

Reply:
That does make sense but we might allow the user to pass a `login` function as well so I kept them separate.

Reply:
We could make a `templateLogin` function that calls both of them.